packages/controller-utils/src/util.ts

```diff
@@ -177,68 +149,6 @@ export function getEtherscanApiUrl(
   return url;
 }
 
-/**
```

Comment:
Moving this code to `controller-utils` created circular dependencies appeared, because this file now relied upon `assets-controllers`, `message-manager`, and `transaction-controller`. To resolve this issue, I moved utility functions that centered around a particular module to that module (for instance, `handleTransactionFetch` is now located within `transaction-controller`).